.github/workflows/on-tag-push-create-release.yml

```
@@ -201,34 +201,70 @@ jobs:
       run: |
         TAG_NAME="${{ needs.common-vars.outputs.tag-name }}"
         cat > ${{ env.RELEASE_DIR }}/RELEASE_NOTES.md << EOF
-        FilePi is a lightweight network file browser for Raspberry Pi and resource-constrained devices.
-        Browse, stream, and manage files from the [Pi View mobile app](https://github.com/renjuashokan/pi_view) or [PiTV explorer Android TV App](https://github.com/renjuashokan/PiTVExplorer).
+        # 🚀 FilePi Server ${TAG_NAME}
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)